BurrowsWheelerTransform/BurrowsWheelerTest/BurrowsWheelerTest.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace BurrowsWheelerTest
+{
```

Comment:
Используйте file-scoped namespaces
```suggestion
﻿namespace BurrowsWheelerTest;
```